code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -16,6 +16,8 @@
 			handle_feeding()
 		if(!stat) // Slimes in stasis don't lose nutrition, don't change mood and don't respond to speech
 			handle_nutrition()
+			if(QDELETED(src)) // Stop if the slime split during handle_nutrition()
```

Comment:
Why is it not stopping the life ticks when it gets qdeleted, this is a bad way to fix the issue

Reply:
It is stopping the ticks, this happens once after split. So it's not stopping the tick currently in progress.

Reply:
Oh is this happening in handle_nutrition?

Reply:
Yeah, that's it. I guess the comment could have been more clear about current tick.